docs/preset-typescript.md

```diff
@@ -34,10 +34,15 @@ npm install --save-dev @babel/preset-typescript
 
 ### With a configuration file (Recommended)
 
+Note that although TypeScript compiler `tsc` actively supports new JavaScript proposals such as optional chaining (`?.`), nullish coalescing (`??`) and class properties (`this.#x`), this preset does not include these features because they are not
```

Comment:
```suggestion
Note that although the TypeScript compiler `tsc` actively supports certain JavaScript proposals such as optional chaining (`?.`), nullish coalescing (`??`) and class properties (`this.#x`), this preset does not include these features because they are not
```